sunpy/net/tests/test_fido.py

```diff
@@ -209,65 +205,36 @@ def test_unifiedresponse_slicing_reverse():
 def test_tables_single_response():
     results = Fido.search(
         a.Time("2012/1/1", "2012/1/5"), a.Instrument.lyra, a.Level.two)
-    tables = results.tables
+
+    with pytest.warns(SunpyDeprecationWarning):
+        tables = results.tables
 
     assert isinstance(tables, list)
     assert isinstance(tables[0], Table)
     assert len(tables) == 1
 
-    columns = ['Start Time', 'End Time', 'Instrument', 'Physobs',
-               'Source', 'Provider', 'Level']
-    assert columns == tables[0].colnames
     assert len(tables[0]) == 5
 
 
 @pytest.mark.remote_data
 def test_tables_multiple_response():
     results = Fido.search(a.Time('2012/3/4', '2012/3/6'),
                           a.Instrument.lyra | (a.Instrument.rhessi & a.Physobs.summary_lightcurve))
-    tables = results.tables
+
+    with pytest.warns(SunpyDeprecationWarning):
+        tables = results.tables
+
     assert isinstance(tables, list)
     assert all(isinstance(t, Table) for t in tables)
     assert len(tables) == 2
 
-    columns0 = ['Start Time', 'End Time', 'Instrument', 'Physobs',
-                'Source', 'Provider', 'Level']
-    columns1 = ['Start Time', 'End Time', 'Instrument', 'Physobs',
-                'Source', 'Provider']
-    assert columns0 == tables[0].colnames and columns1 == tables[1].colnames
-
     assert all(entry == 'LYRA' for entry in tables[0]['Instrument'])
     assert all(entry == 'RHESSI' for entry in tables[1]['Instrument'])
 
 
-@pytest.mark.remote_data
```

Comment:
Why have these tests gone?

Reply:
The tables attribute is deprecated and this test took *a long time* to run, so I dropped it.